### ISSUE TITLE
Can't reverse-mount from Debian to MacOS

### ISSUE THREAD

**Issue by RReverser** [dustymabe/vagrant-sshfs#62]
When trying to change the direction of mount to get client folder in host as per example

```
override.vm.synced_folder "~/...", "/home/vagrant/...", type: "sshfs", reverse: true
```

Getting the following error:

```
==> default: Mounting SSHFS shared folder...
==> default: mounting folder via SSHFS: guestpath:/home/vagrant/... => hostpath:~/...
==> default: Checking Mount..
==> default: Checking Mount..
==> default: Checking Mount..
==> default: Checking Mount..
==> default: Checking Mount..
==> default: Checking Mount..
==> default: Checking Mount..
Mounting SSHFS shared folder via reverse SSHFS mount failed. Please
look at the below output from from the processes that were run.

SSHFS command output:

sh: -o: command not found
```

**Comment by dustymabe** [comment]
it looks like it can't find the sshfs binary as it should have `/path/to/sshfs -o`. What does `which sshfs` return?

**Comment by RReverser** [comment]
@dustymabe You're right, symlinking fixed the problem, although error message is really not very obvious :) Thanks!

**Comment by dustymabe** [comment]
@RReverser well it would be nice to know where `sshfs` is on your machine.. I thought we had a better error message for this but I guess we don't. We should though so feel free to re-open this.

**Comment by RReverser** [comment]
Reopened with updated title.

**Comment by MacDada** [comment]
I have the same error.

`$ which sshfs` – nothing

```
$ sshfs
-bash: sshfs: command not found
```

I installed the plugin with `vagrant plugin install vagrant-sshfs`, but I didn't install `sshfs` separately – should I?

**Comment by dustymabe** [comment]
yes, you have to install sshfs separately

**Comment by MacDada** [comment]
It should be mentioned [in the docs](vagrant plugin install vagrant-sshfs) then ;)

**Comment by dustymabe** [comment]
Installing `sshfs` on the host is only needed if you are doing a reverse mount, which isn't the most common use case. It is somewhat mentioned in https://github.com/dustymabe/vagrant-sshfs/blob/master/README.adoc#options-specific-to-reverse-mounting-guesthost-mount. Feel free to open a PR to help clarify things more.
